src/main/java/net/minecraftforge/fml/common/registry/VillagerRegistry.java

```diff
@@ -212,6 +402,27 @@ public VillagerCareer getCareer(int id)
             }
             return this.careers.get(0);
         }
+        
+        public String getName(){
```

Comment:
No, bouncer methods again bad! Registry objects have a standard method to get the name we want modders to get used to using it.